documentation/asciidoc/accessories/iqaudio/faq.adoc

```diff
@@ -24,24 +23,23 @@ $ sudo alsactl restore -f /home/pi/usecase.state
 === Using external USB devices
 
 If you want to enable 1.2 amp USB support (to allow USB hard disks to power up when
-accessory boards are in use) then you may also want to add the following line to your `/boot/config.txt`
+accessory boards are in use), you may also want to add the following line to your `/boot/config.txt`
 file:
 
 ----
 max_usb_current=1
 ----
 
-Please ensure your Raspberry Pi power supply is able to deliver a minimum of 2.5 amps if you are powering via mini USB and
-enable `max_usb_current`.
+Please ensure your Raspberry Pi power supply is able to deliver a minimum of 2.5 amps if you are powering via mini USB, and enable `max_usb_current`.
```

Comment:
Huh? This probably doesn't belong in this documentation? :thinking:

Reply:
Maaaaybe? Should be a `NOTE:` though, and not main text and have an `xref:` to https://www.raspberrypi.com/documentation/computers/getting-started.html#setting-up-your-raspberry-pi where power supplies are discussed.

Reply:
At the _very_ least, "mini USB" needs changing to "micro USB".

Reply:
Para dropped accordingly.